presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcPreparedStatement.java

```diff
@@ -101,6 +103,46 @@ public void testExecuteQuery()
         }
     }
 
+    @Test
+    public void testGetMetadata()
+            throws Exception
+    {
+        try (Connection connection = createConnection("blackhole", "blackhole")) {
+            try (Statement statement = connection.createStatement()) {
+                statement.execute("CREATE TABLE test_get_metadata (" +
+                        "c_boolean boolean, " +
+                        "c_decimal decimal, " +
+                        "c_decimal_2 decimal(10,3)," +
+                        "c_varchar varchar, " +
+                        "c_varchar_2 varchar(10), " +
+                        "c_row row(x integer, y array(integer)), " +
+                        "c_array array(integer), " +
+                        "c_map map(integer, integer))");
+            }
+
+            List<String> colNames = Arrays.asList("c_boolean", "c_decimal", "c_decimal_2",
```

Comment:
Basically, we don't use abbreviation. I know this class has abbreviations and you probably followed it, though.